docs/_source/_common/snippets/start_page.md

```diff
@@ -1,36 +1,78 @@
-::::{tab-set}
+Welcome to Argilla! 👋😁 We're thrilled to have you on board. Beginning with Argilla is as simple as it gets, and you've successfully taken the first step!
```

Comment:
Maybe "Welcome to Argilla!👋😁 " could be a header and the rest a sub-title?

Reply:
In fact, there was a header before, but  it got removed

Reply:
it was me 😆 it felt too much as a docs page, happy to iterate a bit once we see how it renders, could you share a screenshot Sara?

Reply:
Yes I agree, maybe we can ask @leiyre to have a closer look at this?